### PR TITLE
JBoss and Wildfly Reviewed and Wordsmithed

### PR DESCRIPTION
**Pull request by sez11a**
@justinchoi001 @jhinkey

### CODE REVIEW

**Comment by jrhoun** [comment]
Just pushed this up. Did a lot of article renaming as we discussed on slack.

@sez11a @justinchoi001 - Had to make several fixes to the JBoss EAP and WildFly articles: https://github.com/liferay/liferay-learn/commit/c437478fe81c8b1e3e4e0ce15e17d07ab3b442cb

File: docs/dxp/7.x/en/installation-and-upgrades/installing-liferay/installing-liferay-on-an-application-server/installing-liferay-on-jboss-eap.md [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+            <resource-root path="com.liferay.petra.io.jar" />
+            <resource-root path="com.liferay.petra.lang.jar" />
+            <resource-root path="com.liferay.petra.memory.jar" />
+            <resource-root path="com.liferay.petra.nio.jar" />
+            <resource-root path="com.liferay.petra.process.jar" />
+            <resource-root path="com.liferay.petra.reflect.jar" />
+            <resource-root path="com.liferay.petra.string.jar" />
+            <resource-root path="com.liferay.registry.api.jar" />
+            <resource-root path="hsql.jar" />
+            <resource-root path="[place your database vendor's jar here]" />
```
Comment (sez11a) [inline docs/dxp/7.x/en/installation-and-upgrades/installing-liferay/installing-liferay-on-an-application-server/installing-liferay-on-jboss-eap.md:46]: See line above.

File: docs/dxp/7.x/en/installation-and-upgrades/installing-liferay/installing-liferay-on-an-application-server/installing-liferay-on-jboss-eap.md [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+            <module name="javax.api" />
+            <module name="javax.mail.api" />
+            <module name="javax.servlet.api" />
+            <module name="javax.servlet.jsp.api" />
+            <module name="javax.transaction.api" />
+        </dependencies>
+    </module>
+    ```
+
+   Replace `[place your database vendor's jar here]` with the driver JAR for your database.
```
Comment (sez11a) [inline docs/dxp/7.x/en/installation-and-upgrades/installing-liferay/installing-liferay-on-an-application-server/installing-liferay-on-jboss-eap.md:60]: See line above.

File: docs/dxp/7.x/en/installation-and-upgrades/installing-liferay/installing-liferay-on-an-application-server/installing-liferay-on-jboss-eap.md [hunk 3]
```diff
@@ -0,0 +1,345 @@
...
+1. [Installing dependencies to the application server](#installing-dependencies)
+1. [Configuring the application server for DXP](#configuring-jboss)
+1. [Connect to a Database](#connect-to-a-database)
+1. [Connect to a Mail Server](#connect-to-a-mail-server)
+1. [Deploying the DXP WAR file to the application server](#deploying-dxp)
+
+## Installing Dependencies
+
+1. Create the folder `$JBOSS_HOME/modules/com/liferay/portal/main` if it doesn't exist and extract the JARs from the dependencies ZIP into this folder.
+1. Copy your database vendor's JDBC .jar file to the same location. Please see the [compatibility matrix](https://www.liferay.com/documents/10182/246659966/Liferay+DXP+7.2+Compatibility+Matrix.pdf/ed234765-db47-c4ad-7c82-2acb4c73b0f9) for a list of supported databases.
```
Comment (sez11a) [inline docs/dxp/7.x/en/installation-and-upgrades/installing-liferay/installing-liferay-on-an-application-server/installing-liferay-on-jboss-eap.md:26]: See line above.

**Pull request closed without merging**